lefthook.yml

```
@@ -1,11 +1,22 @@
+post-merge:
+  commands:
+    update-env:
+      run: bunx vercel env pull
+    install-deps:
+      run: bun i
 
 commit-msg:
   commands:
     "lint commit message":
-      run: pnpm commitlint --edit {1}
+      run: bun commitlint --edit {1}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)